tests/unit/test-cmd/test.build.js

```diff
@@ -102,9 +102,8 @@ describe('build', () => {
           .then(makeSureItFails())
           .catch((error) => {
             assert.instanceOf(error, UsageError);
-            assert.match(
-              error.message,
-              /Error .* file .*messages\.json: SyntaxError: Unexpected string/);
+            assert.match(error.message, /Unexpected token '"' at 1:15/);
```

Comment:
we should add an additional assertion here to check that the error message starts with the expected message, e.g. (using '^' in the regular expression):
```js
assert.match(error.message, /^Error parsing messages.json/);
```